stories/Toast.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { Toast } from '../src/components';
+
+export default {
+  title: 'Components/Toast',
+  component: Toast,
+  decorators: [
+    (Story) => (
+      <div style={{ margin: '150px auto', textAlign: 'center' }}>
+        <Story />
+      </div>
+    ),
+  ],
+  argTypes: {
+    placement: {
+      control: { type: 'select' },
+      options: [
+        JSON.stringify({ horizontal: 'left', vertical: 'top'}),
+        JSON.stringify({ horizontal: 'center', vertical: 'top'}),
+        JSON.stringify({ horizontal: 'right', vertical: 'top'}),
+        JSON.stringify({ horizontal: 'left', vertical: 'center'}),
+        JSON.stringify({ horizontal: 'center', vertical: 'center'}),
+        JSON.stringify({ horizontal: 'right', vertical: 'center'}),
+        JSON.stringify({ horizontal: 'left', vertical: 'bottom'}),
+        JSON.stringify({ horizontal: 'center', vertical: 'bottom'}),
+        JSON.stringify({ horizontal: 'right', vertical: 'bottom'})
+      ],
+    },
+  },
+};
+
+const Template = (args) => {
+  const { placement, ...restArgs } = args;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
is this error still appearing? I see that you are using them

Reply:
Thanks, will remove.